plugins/outputs/sematext/sematext.go

```diff
@@ -0,0 +1,150 @@
+package sematext
+
+import (
+	"fmt"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/outputs"
+	"github.com/influxdata/telegraf/plugins/outputs/sematext/processors"
+	"github.com/influxdata/telegraf/plugins/outputs/sematext/sender"
+	"github.com/influxdata/telegraf/plugins/outputs/sematext/serializer"
+	"net/http"
+	"net/url"
+)
+
+const (
+	defaultSematextMetricsReceiverUrl = "https://spm-receiver.sematext.com"
+)
+
+// Sematext struct contains configuration read from Telegraf config and a few runtime objects.
+// We'll use one separate instance of Telegraf for each monitored service. Therefore, token for particular service
+// will be configured on Sematext output level
+type Sematext struct {
+	ReceiverUrl string          `toml:"receiverUrl"`
```

Comment:
I've checked a few other plugins and noticed all use the underscore case for the config properties. Wondering whether we should be consistent in this regard?

Reply:
Yes, you are right, I will adjust.